W-14604521: Update Documentation to Reflect the Required Sequence of Steps Installing AM Agent on Clusters and Server Groups (#261)
mulesoft/docs-monitoring: modules/ROOT/pages/am-installing.adoc

. Follow the instructions for <<install_ap_monitoring_onprem,installing the Anypoint Monitoring agent>> on-premises.
. Restart Mule.

> *Note:* If a runtime is added/removed from the cluster or a server group, the AM agent needs to be reinstalled.

Reviewer: ```suggestion
NOTE: If a runtime is added or removed from the cluster or a server group, you must reinstall the Anypoint Monitoring agent.
```
